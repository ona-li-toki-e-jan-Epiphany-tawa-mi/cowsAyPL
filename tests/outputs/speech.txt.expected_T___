/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| Technology is both the problem and the s |
| olution, our dearest weapon. Unbound by  |
| law and morality, the might of the elect |
| ron and switch propels us ever forward.  |
| Far greater than any magic or sorcery, a |
| nd it is us who wields it.               |
\__________________________________________/
                                        \                  
                                         \                 
                                           ^__^            
                                           (oo)\_______    
                                           (__)\       )\/\
                                            __ ||----w |   
                                               ||     ||   
