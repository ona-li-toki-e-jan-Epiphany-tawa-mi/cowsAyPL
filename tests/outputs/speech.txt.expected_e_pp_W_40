/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| Technology is both t |
| he problem and the s |
| olution, our dearest |
|  weapon. Unbound by  |
| law and morality, th |
| e might of the elect |
| ron and switch prope |
| ls us ever forward.  |
| Far greater than any |
|  magic or sorcery, a |
| nd it is us who wiel |
| ds it.               |
\______________________/
                    \                  
                     \                 
                       ^__^            
                       (pp)\_______    
                       (__)\       )\/\
                           ||----w |   
                           ||     ||   
