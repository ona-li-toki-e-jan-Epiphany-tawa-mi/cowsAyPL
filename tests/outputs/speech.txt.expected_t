/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| + t Technology is both the problem and t |
| he solution, our dearest weapon. Unbound |
|  by law and morality, the might of the e |
| lectron and switch propels us ever forwa |
| rd. Far greater than any magic or sorcer |
| y, and it is us who wields it.           |
\__________________________________________/
                                        \                  
                                         \                 
                                           ^__^            
                                           (oo)\_______    
                                           (__)\       )\/\
                                               ||----w |   
                                               ||     ||   
