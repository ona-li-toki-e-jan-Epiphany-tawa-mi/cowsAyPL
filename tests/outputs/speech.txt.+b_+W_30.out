/¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯\
| Technology is both the problem |
|  and the solution, our dearest |
|  weapon. Unbound by law and mo |
| rality, the might of the elect |
| ron and switch propels us ever |
|  forward. Far greater than any |
|  magic or sorcery, and it is u |
| s who wields it.               |
\________________________________/
                              \                  
                               \                 
                                 ^__^            
                                 (==)\_______    
                                 (__)\       )\/\
                                     ||----w |   
                                     ||     ||   
